source/coverage.cmake

```diff
@@ -0,0 +1,3 @@
+if(YOTTA_CFG_DEBUG_OPTIONS_COVERAGE_MODULES_SOCKETS)
+  set( CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -fprofile-arcs -ftest-coverage" )
```

Comment:
I also don't want ever source file for the target to get those compilation options. I only want the files in this directory to get them.